### PR TITLE
Remove non-alpha storage features from alpha job

### PR DESCRIPTION
**Pull request by msau42**

### CODE REVIEW

File: config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml [hunk 1]
```diff
@@ -344,7 +344,7 @@ presubmits:
         - --provider=gce
         - --runtime-config=api/all=true
         - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gce-alpha-features
-        - --test_args=--ginkgo.focus=\[Feature:(ServiceAccountIssuerDiscovery|StorageVersionHash|Audit|PodPreset|ExpandCSIVolumes|ExpandInUseVolumes|RunAsGroup|TTLAfterFinished|NodeLease|VolumeSnapshotDataSource|CSIInlineVolume|\s*CSIStorageCapacity)\]|Networking --ginkgo.skip=\[Feature:(SCTPConnectivity|Volumes|Networking-Performance)\]|IPv6|csi-hostpath-v0 --minStartupPods=8
```
Comment (cheftako) [inline config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml:347]: Do we run these CSI tests anywhere other than alpha feature test jobs? Do we want it running somewhere else before we remove them from here?

Comment (msau42) [reply config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml:347]: Most are running in the standard ci jobs now that they're beta features. The snapshot tests are running in its own job: https://k8s-testgrid.appspot.com/sig-storage-kubernetes#volume-snapshot

**Pull request merged**